src/plopp/graphics/lineview.py

```diff
@@ -103,7 +103,14 @@ def __init__(
         )
         self.canvas.yscale = norm
 
+        # The line view is potentially plotting many lines. When ``render`` is called,
+        # it calls ``update`` for each line. At the end of ``update``, a range autoscale
+        # is applied. We want to avoid autoscaling at the end of each ``update`` call,
+        # because searching for limits could be expensive.
```

Comment:
For 1d plots, the `LineView.update()` calls autoscale because we want the axis range to update when the values of the points change.

For 2d/3d plots, we are currently not updating the positions of the image or the scatter3d points.
Maybe it's something we should be doing, but it has not been necessary so far.

However, we are calling an `autoscale` on the colormapper (scales the range of the colorbar) on every update (even during `render`) for 2d/3d plots (the call is hidden inside the calls to `self.colormapper.update()`). It just has never really been an issue because we only very rarely plot more than one image, or more than ~5 point clouds on the same axes. So figure creation has never really been an issue.

We could add an `autoscale` argument to `update` which is `True` by default, and then set it to `False` when `render` is calling it. This would disable axis and colorbar autoscaling.